### DISCUSSION TITLE
Consider joining forces with react-bootstrap

### DISCUSSION THREAD

**Discussion by miso-belica** [reactstrap/reactstrap#2151] [General]
Hi, I know that this is maybe a naive idea, but please stay with me :)

In the past, I wanted to use Boostrap in React and found this library along with https://github.com/react-bootstrap/react-bootstrap. I was curious what are the differences there but couldn't find the answer anywhere except for a few API differences that I didn't consider really a difference. So I simply choose one. And the history repeats and I am here again to choose.

I wonder if these 2 libraries could be merged somehow so people don't split the time for two not so different libraries. Did you guys from @reactstrap and @react-bootstrap talk about this? As I said, I am probably naive because there can be technical or political issues with this. But at least maybe some discussion or maybe the common source of the comparison would be great. I found https://github.com/react-bootstrap/react-bootstrap/issues/4625 where one of the points is _(more of a one-shot article 😄) Comparing API differences between React Bootstrap and Reactstrap._ but it's from the year 2019 and the projects are still without the explanation when to choose which one. Joining forces at least to write a good article about this from both sides could be really beneficial IMHO.

Have a nice day :)

**Comment by kyletsang** [comment]
This has been discussed a bit in https://github.com/reactstrap/reactstrap/issues/1748

reactstrap and react-bootstrap work fine and are stable for bootstrap 4.  They're both getting bootstrap 5 releases as well (pre-releases are currently available for both).

IMO, if you're looking to start a new project, I'd go with react-bootstrap because the code is more up to date (TS codebase, hooks, strict mode support, new JSX transform etc).  If you're already on reactstrap and looking to upgrade, sticking with reactstrap would be the easiest option.  

I currently use both, but all my new projects start with react-bootstrap now and we have plans to migrate from reactstrap once bootstrap 5 becomes stable.

**Reply by krnlde** [reply]
Bootstrap 5 is now stable
